README.md

```diff
@@ -67,9 +93,10 @@ git config --global --unset url."http://".insteadOf
 ```
 
 _Notes:_ 
+ - If you are behind NTLM proxy you can use [Cntlm Authentication Proxy](http://cntlm.sourceforge.net/).
  - If you are on Windows replace `export` with `set` to set env variables
- - Replace `localhost:3128` with standard pattern `http://user:pwd@host:port`
- - Git configuration is needed because Bower use it for fetching from GitHub
+ - Replace `localhost:3128` with the standard pattern `http://user:pwd@host:port`
+ - For zeppelin-web: currently there is no way to reach Bower main repo through NTLM proxy
```

Comment:
Yes I have seen this issue and I did some tests with different bowerrc
configurations. I have also reached their online support without success.
Honestly I didn't tried the weird fiddle step, but tomorrow I'll give it a
chance.
Il 26 giu 2016 6:42 PM, "Lee moon soo" <notifications@github.com> ha
scritto:

> In README.md
> <https://github.com/apache/zeppelin/pull/1069#discussion_r68505652>:
>
> >   - If you are on Windows replace `export` with `set` to set env variables
> > - - Replace `localhost:3128` with standard pattern `http://user:pwd@host:port`
> > - - Git configuration is needed because Bower use it for fetching from GitHub
> > + - Replace `localhost:3128` with the standard pattern `http://user:pwd@host:port`
> > + - For zeppelin-web: currently there is no way to reach Bower main repo through NTLM proxy
>
> Thanks @fvaleri <https://github.com/fvaleri> for the documentation!
> One question, is it really true that bower can not go through NTLM proxy?
> I've found a related issue bower/bower#1368
> <https://github.com/bower/bower/issues/1368>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/zeppelin/pull/1069/files/065e3f399ad39e315560f676ba8d570eab106627#r68505652>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK7P8t1XgnYtJhx1SvfkVr9DHIc1ukgaks5qPqvZgaJpZM4I8OF5>
> .
>